### DISCUSSION TITLE
Add configurable path/filename to secondary metadata source

### DISCUSSION THREAD

**Discussion by qiyundai** [adobe/helix-home#352] [Ideas]
The context: At Adobe Express, we have large batches of pages that share the same page structure & block configuration with different content. With the metadata combination feature originally discussed [here](https://github.com/adobe/helix-home/discussions/336), we were able to optimize and maintain our templates/ pages with much ease.

Now, as we add the colors pages, which is another category of large amount of SEO pages using the same default.docx + default/metadata.xlsx structure, we are starting to manage 3 different metadata.xlsx files. It's been flagged by our authors that it starts to get confusing for them. Sometimes they have a hard time realizing they are looking a metadata file that's not the one they are looking for.

The ask: Instead of just the implied metadata location, would it be possible to add a way in the fstab file for use to designate a different file other than the (`${resource-path}/metadata.json`) to be the metadata source?

Thanks!

**Comment by tripodsan** [comment]
please note that there are 2 types of metadata sheet mechanisms supported:
1. the metadata based on the folder mapping, i.e. virtual pages below a folder get the metadata of the folder applied. I believe that is what you are referring to above.
2. global metadata, which defaults on `/metadata.json` but can also be extended to support multiple metadata sheets, configured in config.xslx.

see  https://www.hlx.live/docs/bulk-metadata#additional-metadata

**Reply by qiyundai** [reply]
You are right! We are trying to explore the possibility to be able to rename the metadata sheet file in the virtual pages root folder.

**Comment by rofe** [comment]
We did discuss an object structure for the mapping definition with freedom to specify the path to the metadata sheet [here](https://github.com/adobe/helix-home/discussions/336#discussioncomment-5591609), but decided to start with the simplest possible solution with an automatically derived path to the metadata sheet.

Looks like we reached a point where we need to add support for both. @tripodsan @davidnuescheler wdyt?

**Reply by tripodsan** [reply]
yes, as suggested:

```
mountpoints:
  /: ......
folders:
  /products/de: 
    resource: /products/default
    metadata: /products/default/metadata-de.json
  /products/fr: /products/default
    resource: /products/default
    metadata: /products/default/metadata-fr.json
```

also, we might want to move the folder mapping from the fstab into the config eventually....maybe for helix5
